Readme.md

```diff
@@ -78,3 +78,196 @@ force any process with names `geth` or `op-*` to shut down.
+proposer = "0xb846C69FA1f6D2DC86Ee44553f67Bbb86e007d08"
+# should it be a separate address?
+challenger = "0x2611596AA00F8438d75f1daF893CF264366fc668"
+```
+
+After this, all the steps to deploy contracts on the L1 follow:
+
+```bash
+op-deployer bootstrap proxy --l1-rpc-url=http://127.0.0.1:58138 --private-key=0x3ae90739336cd848513adb4a5d6cae372b64135fb4d214aa1a25948a21c7b7fd  --proxy-owner 0x2611596AA00F8438d75f1daF893CF264366fc668 --outfile proxy-output.json
+
+op-deployer bootstrap superchain --l1-rpc-url=http://127.0.0.1:58138 --private-key=0x3ae90739336cd848513adb4a5d6cae372b64135fb4d214aa1a25948a21c7b7fd  --superchain-proxy-admin-owner 0x2611596AA00F8438d75f1daF893CF264366fc668 --protocol-versions-owner 0x2611596AA00F8438d75f1daF893CF264366fc668 --guardian 0x2611596AA00F8438d75f1daF893CF264366fc668  --outfile superchain-output.json
+
+op-deployer bootstrap implementations --l1-rpc-url=http://127.0.0.1:58138 --private-key=0x3ae90739336cd848513adb4a5d6cae372b64135fb4d214aa1a25948a21c7b7fd    --outfile bootstrap_implementations.json --superchain-config-proxy 0x3000f5214879c31a203d5df693f7e7c2850d52f5   --protocol-versions-proxy  0x45af18faf3ec5fa0401cded0e59c21377adc689e     --upgrade-controller 0x2611596AA00F8438d75f1daF893CF264366fc668  --challenger 0x2611596AA00F8438d75f1daF893CF264366fc668 --superchain-proxy-admin 0x481733903ad24403dd6dabc00ce4db6789a0c41f
+```
+
+Most of the addresses/keys are the usual admin ones all over. Of note here is the final step only, as it needs other addresses as well.
+Protocol versions, superchain config proxy and superchain proxy admin come from the superchain bootstrapping file, i.e. `superchain-output.json`:
+
+```json
+ {
+   "protocolVersionsImplAddress": "0x37e15e4d6dffa9e5e320ee1ec036922e563cb76c",
+   "protocolVersionsProxyAddress": "0x45af18faf3ec5fa0401cded0e59c21377adc689e",
+   "superchainConfigImplAddress": "0xce28685eb204186b557133766eca00334eb441e4",
+   "superchainConfigProxyAddress": "0x3000f5214879c31a203d5df693f7e7c2850d52f5",
+   "proxyAdminAddress": "0x481733903ad24403dd6dabc00ce4db6789a0c41f"
+ }
+ 
+```
+
+Next, the deployer needs to apply the actual intent:
+
+```bash
+op-deployer apply --l1-rpc-url=http://127.0.0.1:58138 --private-key=0x3ae90739336cd848513adb4a5d6cae372b64135fb4d214aa1a25948a21c7b7fd
+```
+
+As noted before, this only seems to work after the L1 safe head has advanced quite a bit, i.e. when geth has produced about 150 blocks.
+
+Now we can generate the genesis and rollup files:
+
+```bash
+op-deployer inspect rollup 42069 > rollup.json
+op-deployer inspect genesis 42069 > genesis.json
+```
+
+These are need to run `op-node` and `op-move` respectively. Some other useful addresses (e.g. all L1 deployments)
+can be generated with other `op-deployer inspect ...` subcommands.
+
+The updated commands to run the components are:
+
+```bash
+cargo r --bin op-move --features storage,op-upgrade --release -- --genesis.l2-contract-genesis genesis.json --auth.jwt-secret jwt.txt
```

Comment:
When the `op-move` is run, it shows the `initial_state_root` for the genesis config. Can we explain that we need to copy the generated `left` value into the default `GenesisConfig` value in `genesis/src/config.rs`. Then we need to delete the `db` folder with and re-run `op-move`.